runtime/doc/todo.txt

```diff
@@ -55,32 +55,14 @@ When a help item can't be found, then open 'helpfile'.  Search for the tag in
 that file and gtive E149 only when not found.  Helps for a tiny Vim installed
 without all the help files.
 
-SpellCap highlight not updated - PR #12428
-
 Virtual text problems:
--  If 'list' is on, 'below' virtual text which includes 1 or 2 characters are
-   gone (Issue #12028)
 -  Virtual text aligned "above": Wrong indentation when using tabs  (Issue
    #12232)
--  Virtual text to the right of the line that isn't visible doesn't cause the
-   'extends' character to show in 'list' mode.  #12478
--  Virtual text to the right of the line that fits exactly on screen causes
-   that line and all lines below it not to be displayed (Issue #12213)
--  Window screen gets wrong when a virtual text is placed on 'above' or
-   'below' on an empty line (Issue #11959)
 -  truncated Virtual text below an empty line causes display error #12493
 
 When 'virtualedit' is "all" and 'cursorcolumn' is set, the wrong column may be
 highlighted. (van-de-bugger, 2018 Jan 23, #2576)
 
-Improve profiling by caching matching functions:  PR  #12192
-
-With 'smoothscroll' set and "lastline" in 'display', moving the cursor to a
-wrapping line that makes the display scroll up may scroll much more than
-needed, thus jump-scrolling. (part of issue 12411)
```

Comment:
This todo item refers to the last comment in issue #12411, which is as of yet unresolved.